File: 1sem/Python/Lista2/Satoshi/Ex01.py
def main():
    vetor:int = [0]*50
    somai:int = 0
    j:int
    contm:int = 0
    med:int = 0

    for j in range(50):
        vetor[j] = int(input(f'Digite o termo {j+1}: '))
        if (vetor[j]%2==1):
            somai += vetor[j]
        if (vetor[j] >= 10 and vetor[j] <= 200):
            med += vetor[j]
            contm += 1

    print(f'Soma dos valores impares: {somai}')
    print(f'Média dos valores entre 10 e 200: {med/contm:.2f}')

if __name__ == '__main__':
    main()

File: 2sem/Java/Lista 2/prj_java10-09-2026/src/Ex01.java
import javax.swing.JOptionPane;
public class Ex01 {
    static int media=0;
    static int imp=0, num=1;
    public static void main (String args[]){
        int vet[] = new int[50];
        
        for (int i=0;i<50;i++){
            vet[i] = Integer.parseInt(JOptionPane.showInputDialog("Digite o valor para a posição "+(i+1)+": "));
            
            if (vet[i] >= 10 && vet[i] <= 200){
                Med10200(vet[i]);
            }
            if (vet[i]%2==1){
                Impares(vet[i]);
            }
        }
        
        System.out.println("Média dos valores entre 10 e 200: "+(float)media/num);
        System.out.println("Soma dos ímpares: "+imp);
    }
    
    static void Med10200 (int n){
        media += n;
        num++;
    }
    
    static void Impares (int n){
        imp += n;
    }
}
